src/Utils/OrientationUtils.js

```diff
@@ -173,6 +202,26 @@ export default {
         };
     },
 
+    /**
+     * Set the quaternion to correct for the
+     * <a href="https://geodesie.ign.fr/contenu/fichiers/documentation/algorithmes/alg0060.pdf">meridian convergence</a>
+     * of the East North Up (ENU) frame to the Lambert Conformal Conic (LCC) frame.
+     * This is a generally small rotation around Z.
+     *
+     * @param {Projection} proj the lcc projection (may be parsed using proj4)
+     * @param {number} proj.lat0 - the latitude of origin
+     * @param {number} proj.long0 - the longitude of the central meridian
+     * @param {Coordinates} [coordinates]  coordinates the origin of the local East North Up
+     * (ENU) frame
+     * @param {THREE.Quaternion} [target=new THREE.Quaternion()] output Quaternion
+     * @return {FunctionOrQuaternion} The target quaternion if coordinates is defined, otherwise, a function to compute it from coordinates.
+    */
+    quaternionToLCC(proj, coordinates, target = new THREE.Quaternion()) {
```

Comment:
The parameter `proj` may not be useless if we use the `crs coordinates` to find it.

Reply:
do you mean "may be useless"?
In general, I do not want to require that coordinates is in the proj CRS.

Reply:
you could write 
`quaternionFromCoordinate(coord)` and use `coords.crs` to map with `Proj4.projection`
and if  the coordinates isn't in the proj CRS:
`quaternionFromCoordinate(coord.as('EPSG:XXXX'))`